fix: allow embedding two pages generated into the same page in "embedded" mode (#9610)
sveltejs/kit: .changeset/late-toes-cry.md

---
'@sveltejs/kit': patch
---

> fix: namespace collision on global object when two pages generated in "embedded" mode are embedded into the same page

Reviewer: maybe we could shorten this a bit
```suggestion
fix: allow embedding two pages generated into the same page in "embedded" mode
```

> fixed the generated namespace for "embedded" sveltekit apps so that two separate pages can be inlined into the same page without clobbering each other

Reviewer: fix messages in the changelog should start with `fix: ` for consistency: https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md

sveltekit should be capitalized as SvelteKit
